src/Honeybee_ExportEPC.py

```diff
@@ -127,18 +140,48 @@ def main(HBZones):
     # return surface data
     return srfData
     # done!
-
-
+#print srfData
 srfTypeDict = {0: "Wall", 1 : "Roof", 2: "Floor", 5 : "Glazing"}
 directionDict = {0: "N", 1 : "NW", 2: "W", 3: "SW", 4 : "S", 5 : "SE", 6 : "E", 7 : "NE"}
 
+
```

Comment:
I can only take two glass types and two wall types. If a wall is below
grade I have an input for that. Currently the code collapses all the walls
down to one type as you noted. I will adapt the next version to take more
than one type.

Thanks!

Patrick Chopson, LEED AP BD+C
MS High Performance Buildings
Georgia Institute of Technology
pchopson3@gatech.edu





On Sat, Feb 14, 2015 at 11:14 AM, Mostapha Sadeghipour Roudsari <
notifications@github.com> wrote:

> In src/Honeybee_ExportEPC.py
> <https://github.com/mostaphaRoudsari/Honeybee/pull/248#discussion_r24715196>
> :
>
> >  srfTypeDict = {0: "Wall", 1 : "Roof", 2: "Floor", 5 : "Glazing"}
> >  directionDict = {0: "N", 1 : "NW", 2: "W", 3: "SW", 4 : "S", 5 : "SE", 6 : "E", 7 : "NE"}
> >
> > +
>
> @pchopson <https://github.com/pchopson> I checked the code again and now
> I'm confused about how you really need the data. Why are you adding up the
> area of surfaces with different constructions back together? Don't you need
> the area separated for each construction? If that is not the case I can
> simplify the initial dictionary so you don't have to do it down there again.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/mostaphaRoudsari/Honeybee/pull/248/files#r24715196>.
>